src/App.js

```diff
@@ -3,9 +3,9 @@ import "./App.css";
 import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
 
 import { ThemeProvider } from "styled-components";
-import { useDarkMode } from "./components/useDarkMode";
-import { lightTheme, darkTheme } from "./components/themes";
-import { GlobalStyles } from "./components/globalStyles";
+import { useDarkMode } from "./Themes/useDarkMode";
+import { lightTheme, darkTheme } from "./Themes/themes";
+import { GlobalStyles } from "./Themes/globalStyles";
```

Comment:
@Justinnn07 why the name of the folder was changed?

Reply:
> @Justinnn07 why the name of the folder was changed?

Themes are not under components right ? so, thought of making a folder for themes